.github/workflows/weekly.yml

```diff
@@ -1,6 +1,7 @@
 name: Weekly package builds
 
 on:
+  pull_request:
   workflow_dispatch:
   # It seems that the push trigger below makes the weekly workflow run on every push
```

Comment:
It seems like these comments are not from this PR. 
Should we uncomment these lines?
Also, I think we can specify this file in the pull request trigger as well.

Reply:
>  Should we uncomment these lines?

I haven't tested if it's still an issue.

>  Also, I think we can specify this file in the pull request trigger as well.

It will spin up 16 jobs, and we should be a bit more conservative there I guess?